### PR TITLE
CBO-587: Transfer fees

### PR DESCRIPTION
**Pull request by cleargif**
#### What
Implement new auto complete component

#### Ticket

https://dsdmoj.atlassian.net/browse/CBO-587

#### Why
DAC 

#### How
Implement a page controller and binding events accordingly

### CODE REVIEW

**Comment by colinbruce** [comment]
This seems to have all the disbursement code in too... 🤔

File: features/page_objects/litigator_transfer_claim_form_page.rb [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 require_relative 'litigator_claim_form_page'
+require_relative 'sections/common_autocomplete_section'
 
 class LitigatorTransferClaimFormPage < LitigatorClaimFormPage
   set_url "/litigators/transfer_claims/new"
 
   section :transfer_detail, LgfsTransferDetailSection, "#transfer-detail"
   section :transfer_fee, LgfsTransferFeeSection, "#transfer-fee"
+  section :auto_one, CommonAutocomplete, "#cc-transfer-stage"
+  section :auto_two, CommonAutocomplete, "#cc-case-conclusion"
```
Comment (colinbruce) [inline features/page_objects/litigator_transfer_claim_form_page.rb]: For clarity could these just be...
```ruby
section :transfer_stage, CommonAutocomplete, "#cc-transfer-stage"
section :case_conclusion, CommonAutocomplete, "#cc-case-conclusion"
```
?
It should make future debugging easier

File: features/step_definitions/litigator_transfer_claim_steps.rb [hunk 2]
```diff
@@ -13,15 +13,15 @@
 end
 
 And(/^I select the transfer stage '(.*)'$/) do |name|
-  @litigator_transfer_claim_form_page.transfer_detail.select_transfer_stage(name)
+  @litigator_transfer_claim_form_page.auto_one.choose_autocomplete_option(name)
```
Comment (colinbruce) [inline features/step_definitions/litigator_transfer_claim_steps.rb]: See rename comment above

**Review by colinbruce — APPROVED** [review]
some minor points, plus the previous disbursement ones, but otherwise :+1:

File: features/page_objects/litigator_transfer_claim_form_page.rb (hunk 1 above)
Comment (cleargif) [reply features/page_objects/litigator_transfer_claim_form_page.rb]: arg.. I changed the IDs and never the name.. will do

**Pull request merged**